### DISCUSSION TITLE
Doesn't Work with RightInbox

### DISCUSSION THREAD

**Discussion by arokyne** [furybee/chrome-tab-modifier#407] [Q&A]
Hi, I DESPERATELY want to use chrome-tab-modifier but it seems to disable my [Right Inbox extension](https://chromewebstore.google.com/detail/rightinbox-email-reminder/mflnemhkomgploogccdmcloekbloobgb?hl=en). Right Inbox is something we cannot get rid of so that is not an option.

Is anyone aware of this issue or have a solution? Thank you!!!!

PS> Right Inbox support doesn't have a resolution and suggested I reach out here.

Thanks!

**Comment by sylouuu** [comment]
Hi @arokyne

Open an issue instead and complete the issue template correctly if you want to have an answer. 

Bests
